pyincore/analyses/joplinempiricalbuildingrestoration/joplinempiricalbuildingrestoration.py

```diff
@@ -170,7 +170,10 @@ def get_spec(self):
                              "ergo:buildingDamageVer6",
                              "ergo:buildingInventory",
                              "ergo:nsBuildingInventoryDamage",
-                             "ergo:nsBuildingInventoryDamageVer2"]
+                             "ergo:nsBuildingInventoryDamageVer2",
+                             'ergo:nsBuildingInventoryDamageVer3',
+                             'ergo:nsBuildingInventoryDamageVer4',
```

Comment:
Just a nitpick, It might look better if the comma at the end of the list element goes away? Same in line 70 as well. But not a big deal. I don't know too much of the content itself but code looks okay. Approve

Reply:
removed. thanks!